### ISSUE TITLE
[Question]: ragflow embedded does not have document reference

### ISSUE THREAD

**Issue by saineshwar** [infiniflow/ragflow#1814]
### Describe your problem

In the Chat Web App "**Embedded**" I am unable to find reference documents and also information icons. Please can you guide me how to enable it.

![image](https://github.com/user-attachments/assets/5d07dffb-9fb2-4fb6-9a6e-8cb2529803ce)

![image](https://github.com/user-attachments/assets/9f9d6871-4a83-43ce-b279-0b445dc4e4ef)

**Comment by KevinHuSh** [comment]
There is a parameter called 'cite', set it 'true'.

**Comment by saineshwar** [comment]
> There is a parameter called 'cite', set it 'true'.
Thank you @KevinHuSh.  In which file I need to set please guide me.

**Comment by KevinHuSh** [comment]
Sorry, my bad. 
It can not support cites yet.

**Comment by zhangdengp** [comment]
I have the same problem, response is [].

**Comment by homegrownhrbs** [comment]
I have the same issue

**Comment by quetzalism** [comment]
Yeah, I have the whole program up and ready to go but I am not interested in implementing it  until the embedded assistant can provide sources like the integrated chat can.

100% important for my usecase of this program

**Comment by windyzhu3** [comment]
same issue exists when i use preview model,  Is these any param can solve the problem?

**Comment by vst212** [comment]
Any plan on this features? Many thanks!
